src/handler.ts

```diff
@@ -17,13 +23,18 @@ const handler = async (request: Request, context: any) => {
   // @ts-ignore Deno global is available at runtime
   ;(globalThis as any).Oxygen.env = Deno.env.toObject()
 
+  // Netlify Edge Functions currently do not support CacheStorage, so we fall back to in-memory caching until they do.
+  const cache =
```

Comment:
Maybe I'm misunderstanding something, but will the in-memory cache persist between calls to the Edge function?

Reply:
Yes, Deno reuses the isolate if possible, so while it will only be cached in one edge node for a limited time, it's better than nothing.